atomic_reactor/schemas/container.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+
+  "title": "container image build configuration",
+
+  "type": "object",
+  "properties": {
+    "platforms": {
+      "type": "object",
+      "properties": {
+        "only": {
+          "$ref": "#/definitions/platforms"
+        },
+        "not": {
+          "$ref": "#/definitions/platforms"
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "additionalProperties": false,
```

Comment:
TBD: Should this be `true` (or omitted)?

Reply:
Removing `additionalProperties: false`:
:heavy_plus_sign:  would allow container.yaml to hold user specific data (which might lead to conflicts with future a-r changes :heavy_minus_sign:)
:heavy_minus_sign: it's easy to forget to update the schema with new properties when needed.

Keeping `additionalProperties: false`:
:heavy_plus_sign: would help users in finding typos
:heavy_plus_sign: schema will have to be updated when new properties are introduced.

Reply:
Let's keep it then. Thanks for analyzing it.